### PR TITLE
update HTTP and MbedTLS compatibilities

### PR DESCRIPTION
**Pull request by snisher**
Haven't tested locally...

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -14,10 +14,10 @@ MsgPack = "99f44e22-a591-53d1-9472-aa23ef4bd671"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 
 [compat]
-HTTP = "0.8"
+HTTP = "1"
```
Comment (mattBrzezinski) [inline Project.toml:17]: https://github.com/JuliaWeb/HTTP.jl/releases

**Comment by snisher** [comment]
Works for me locally.

**Comment by mattBrzezinski** [comment]
These changes will no longer allow Julia 1.0 to work with this package. Although it's not a particularly used package AFAIK I think we should still be supporting the Julia LTS version. We can use `HTTP@0.9`, but keep `MbedTLS@0.6`.

Unless there is a strong argument here for upgrading `MbedTLS`, I think keep this as is until the next LTS comes out (soonTM) keeps this package available to as many people as possible.

**Comment by snisher** [comment]
Makes sense. MbedTLS < 1.0 has a build script so isn't easily compatible with compiling an app with PackageCompiler, but I can use my local version until the new LTS. Thanks!

**Comment by YoungFaithful** [comment]
I would love to see the update HTTP@0.9 be shipped. Should I add a new merge request with it?

**Comment by nlw0** [comment]
Since all these dependencies have been upgraded in other PRs, I'm closing this.

**Pull request closed without merging**